pyannote/audio/transforms/transforms.py

```diff
@@ -0,0 +1,26 @@
+from typing import List
+
+import torch
+import torchaudio
+from torch import Tensor
+from torch_audiomentations.core.transforms_interface import BaseWaveformTransform
+
+
+class Reverb(BaseWaveformTransform):
```

Comment:
This should be part of `torch_audiomentations`.

Reply:
Yes I want to add this but a batch version https://github.com/asteroid-team/torch-audiomentations/issues/38. I thought for now we could just use this as you said you were interested in experimenting with it.

Reply:
I have contributed `ApplyImpulseResponse` to `torch_audiomentations` and started using it already with `EchoThief` real reverb recording (http://www.echothief.com/)